Remove svg from the default sent image types

janschulz opened the pull request:
SVG currently has a few problems:

* complicated plots can lead to very big streams sent which takes ages to show up in the browser and lead to bad scrolling behaviour
* svg output in the browser is not saveable in chrome (as it is isolated into an iframe)
* using two plot formats also means that two plots have to be drawn which takes twice as long.

Closes: https://github.com/IRkernel/IRdisplay/issues/13

flying-sheep commented:
i’m OK with merging that, but please stay factual:

* not complicated, but *broken* (e.g. that huge ggplot object) or *useless* plots do. useless plots have e.g. huge amounts of points that are on top of each other and therefore and give no idea of density
* the fact that they’re not savable is 1. preventable (#349) and 2. not limited to SVG
* the time it takes is OK

the real problem is that other tech have too many bugs regarding SVG so it’s pragmatic to remove it.

flying-sheep commented:
jup, you need to adapt the tests.

janschulz commented:
Sorry for the missed (failing) tests, fixed now, hopefully.

I don't agree on point 1 and 3 but didn't want to start arguing again with you, so I just removed the commit message and pointed them here and the original issue.

flying-sheep commented on test_ir.py:
```diff
@@ -88,19 +88,16 @@ def test_irkernel_plots(self):
         self.assertIn('image/png', data)
         
-        # we isolate only svg plots
+        # we isolate only svg plots, which are not included anymore
```
please add a test checking if they’re isolated when configured to be there

janschulz replied:
done, sorry for missing that. :-/

janschulz commented:
so, finally got the errors out... Apart from the osx build, which fails in installing latex?

flying-sheep commented:
merged: 8cdfe5e001e6ef746babcdfea5a6bf1692df9678
